handbook: Update how to close a deal (#3456)
FlowFuse/website: src/handbook/sales/engagements.md

Important Note: Verbal agreements, informal emails, or other non-binding expressions of interest do not constitute a closed deal. The legal commitment, as evidenced by a PO or signed quote, is the sole determinant.

### Steps 

1. Setting the deal when either the quote is signed or a PO is Issued.
  * Check if the order form is signed and that the quote has all the right information.

>   * Describe additional information in line items
>   * Set line items to annual and payment terms to payment renew until cancelled
>   * Deal amount should be equal to ACV (annual contract value)

Reviewer: ```suggestion
  * Deal amount should be equal to IACV (incremental annual contract value)
```

cc @MichaelBDavis -- Please update Hubspot accordingly

1. Upload the documents to the Google Drive in the correct directory
   * [Signed quotes and P.O.'s](https://drive.google.com/drive/folders/1Nb3UqFiE56ymgQnyfkDKHMAe6L3akNzQ)
   * If negotiated custom, the [Subscription agreement](https://drive.google.com/drive/folders/1h6jBvkJ2oDrBL7jw751RfAzBKkkprKDc)
1. Update the deal to:
   * Set the Renewal data for the deal

>    * Move the deal to `Closed Won` in HubSpots Deal overview, update the close date to when the customer committed to purchasing.

Author: ```suggestion
   * Move the deal to `Closing` in HubSpots Deal overview, update the close date to when the customer committed to purchasing.
```

> 1. Send an email to customer to introduce their Customer Success manager.

Author: asking for PO, Introduction CSM to Customer ;  Set meeting cadence right away - every month 30 minutes. Quarterly business reviews

> 1. Send an email to customer to introduce their Customer Success manager.

Author: ```suggestion
1. Send an email to customer to set meeting monthly cadence, quarterly. business review and to introduce their Customer Success manager, 
```

> 1. Send email to CSM to handover new customer, inlcuding the name of the champion, contract start date.

Author: yes that can be put together. 
asking for PO, Introduction CSM to Customer ;  Set meeting cadence right away - every month 30 minutes. Quarterly business reviews
